README.md

```diff
@@ -46,12 +46,12 @@ Once conda finishes creating the virtual environment, activate `dsstdeface`.
 conda activate dsstdeface
 ```
 
-## Using `dsst_defacing_wf.py`
+## Usage
 
 To deface anatomical scans in the dataset, run the `src/dsst_defacing_wf.py` script. From within the `dsst-defacing-pipeline` cloned directory, run the following command to see the help message.
 
 ```text
-% python src/dsst_defacing_wf.py -h
+% python src/run.py -h
```

Comment:
incredibly minor piece of feedback, but for more obviousness, I would be inclined to put the run.py at the root of the repository folder structure in the parent of `src/`. Not necessary to accept this PR.

Reply:
Option 1 and Option 2 below still have `src/dsst_defacing_wf.py` instead of `src/run.py`.

Reply:
It was a quick fix so I fixed and committed it to this branch

Reply:
thanks for catching those!